src/internal/packager/prepare.go

```diff
@@ -39,6 +39,10 @@ func FindImages(baseDir, repoHelmChartPath string) {
 		message.Fatal(err, "Unable to read the zarf.yaml file")
 	}
 
+	if err := config.SetActiveVariables(config.ZarfYAML, false); err != nil {
```

Comment:
Should we add the `--set` flag to the `zarf prepare find-images` command? If the images are variable'd out it might get missed if there is no default value.

Reply:
I tested this and asked about it here:  https://github.com/defenseunicorns/zarf/issues/613 What happens is it will print out the variable name (i.e. `###ZARF_AGENT_IMAGE###`).  `zarf prepare find-images` doesn't template the components (didn't before when it was just `AGENT_TAG`) so setting anything here has no effect on output without changing that.  The main reason variables are needed here though is if you variablize imports for components (like I did in the `examples/composable-package-variables` case)

Reply:
(though I suppose it would be useful if you then `--set` those variablized imports to something else)

Reply:
And added